packages/desktop-client/src/components/reports/spreadsheets/grouped-spreadsheet.tsx

```diff
@@ -0,0 +1,136 @@
+import { runQuery } from 'loot-core/src/client/query-helpers';
+import { send } from 'loot-core/src/platform/client/fetch';
+import * as monthUtils from 'loot-core/src/shared/months';
+import { integerToAmount } from 'loot-core/src/shared/util';
+
+import { categoryLists } from '../ReportOptions';
+
+import filterHiddenItems from './filterHiddenItems';
+import makeQuery from './makeQuery';
+import recalculate from './recalculate';
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏this should also be `.ts`